src/layouts/BlogPosts.astro

```diff
@@ -62,7 +62,7 @@ const {
 			posts.map((post) => (
 				<PostCard
 					variant="large"
-					url={`${BLOG_URL}${post.url}`}
+					url={post.url}
```

Comment:
No it's a relative URL, and that's what you want here I think!

Reply:
My concern is that we have relative URLs like: `./2024/10/some-post` and that will break some pages like the tags pages. For example, if you go to `/tags/web-development` and have the relative URL post, it will not work as expected.

Reply:
Oh no, the URLs start with a slash! So that should be good

Reply:
I just checked, the urls don't start with slashes... 😕

<img width="648" alt="a screenshot of a terminal logging urls, showing all of them start with a number like 2024..., etc." src="https://github.com/user-attachments/assets/f93efc39-6558-465b-b4ab-8c3c092c153d">

Reply:
Here is an example of an url on the tags page:

<img width="160" alt="url showing: https://localhost:3000/tags/2024/09/new-blog" src="https://github.com/user-attachments/assets/7350b401-9b23-40d5-b271-60492ae021be">

Reply:
Oof sorry, I must have looked at this wrong. Will fix this PR!

Reply:
No problem, I just fixed the issue on the root. Take a look at this branch:

https://github.com/torontojs/blog/compare/relative-uris...fix/post-urls